### DISCUSSION TITLE
[预览] v2.1.3

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2976] [分享 / Share]
除了 `v2.1.3` 的内容以外, 本预览版中还包括:
- 支持 AV1 编码下载. (#2941)
- `自动更新器` 的选项中, 可以手动触发更新检查.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.3-preview'>[预览] v2.1.3</a>.</em>

**Comment by Geeyun-JY3** [comment]
issue form 的 `是否开启 HEVC` 部分该跟着改成 `播放策略` 了。
